tex/H113/structure.tex

```diff
@@ -1,7 +1,7 @@
 \chapter{The PID structure theorem}
 The main point of this chapter is to discuss a classification
 theorem for finitely generated abelian groups.
-This won't take long to do, and if you like you can read
+This won't take long to do, and if you like, you can read
```

Comment:
Maybe this isn't technically a typo? But it reads so strangely without it.

Reply:
I think it's better with the comma, yes.